build/common/installer/scripts/tomlparser.rb

```diff
@@ -267,12 +267,12 @@ def populateSettingValuesFromConfigMap(parsedConfig)
 =end
 
 def get_command_windows(env_variable_name, env_variable_value)
-  return "[System.Environment]::SetEnvironmentVariable(\"#{env_variable_name}\", \"#{env_variable_value}\", \"Process\")" + "\n" + "[System.Environment]::SetEnvironmentVariable(\"#{env_variable_name}\", \"#{env_variable_value}\", \"Machine\")" + "\n"
+  "#{env_variable_name}=#{env_variable_value}\n"
```

Comment:
no return statement? is this a ruby paradigm?

Reply:
yes In Ruby, the last evaluated expression in a method is returned by default, but for clarity I can make it explicit